### PR TITLE
RHN cell

### PR DESCRIPTION
**Pull request by abhaikollara**

### CODE REVIEW

**Comment by farizrahman4u** [comment]
Update example to use the RHN cell. Also, leave an empty line at the end of file (pep8).

**Comment by farizrahman4u** [comment]
Specify output shape for Lambda layers to avoid warnings on theano backend. `output_shape=lambda s: s`

**Comment by farizrahman4u** [comment]
Update?

File: recurrentshop/advanced_cells.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                        use_bias=self.use_bias,
+                        bias_initializer=self.bias_initializer,
+                        bias_regularizer=self.bias_regularizer,
+                        bias_constraint=self.bias_constraint)(ht)
+
+            cli = Lambda(lambda x: 1.0 - x, output_shape=lambda s: s)(tli)
+            ht = add([multiply([hli, tli]), multiply([ht, cli])])
+
+        return Model([x, h_tm1], [ht, Identity()(ht)])
+
```
Comment (farizrahman4u) [inline recurrentshop/advanced_cells.py]: New line at end of file.

**Comment by abhaikollara** [comment]
@farizrahman4u Modifying the example to use RHN cell would mean rewriting the RHN cell to include variational dropout, which is a regularization technique rather than a part of the RHN architecture itself.

**Comment by farizrahman4u** [comment]
In the example, simply call the RHNCell to do the computation. Nothing has to be changed on the RHNCell side.

**Pull request merged**